src/site/styles/_our-community.scss

```diff
@@ -0,0 +1,44 @@
+@import "vars";
+
+.our-community {
+
+  .row {
+    padding: 5vh 0;
+
+    &:first-child {
+      padding-top: 0;
+    }
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $brand-dark;
+    }
+  }
+
+  .icon-container {
+    display: flex;
+
+    .icon {
+      width: 75%;
+      height: 100%;
+      fill: $brand-dark;
+    }
+  }
+
+  .page-hero {
+    background: url(/assets/our_community_hero_background.png), lighten($brand-dark, 40);
+    background-blend-mode: multiply;
+    background-position: center bottom;
+    background-size: 100% auto;
+    @include media-breakpoint-down(sm) {
+      background-size: auto 100%;
+    }
```

Comment:
I pulled one more thread here.. added a breakpoint so the hero image doesn't tile vertically on small screens.

Reply:
Still getting a bit of tiling. try setting the background size to be `background-size: cover;` for all screen sizes. The about page has this issue too, so if we can potentially share a style here, that would be good.


![image](https://user-images.githubusercontent.com/9632520/40889572-73c15384-672e-11e8-8399-e452c5f20a7f.png)

Reply:
fixed

This is witchcraft -- glad I know someone who has listed CSS as a skill on their linked in profile!